source/MaterialXContrib/MaterialXNode/MaterialXData.cpp

```diff
@@ -107,17 +129,28 @@ void MaterialXData::registerFragments()
 	{
         if (MHWRender::MFragmentManager* fragmentMgr = theRenderer->getFragmentManager())
 		{
-			const bool fragmentExists = (glslFragmentWrapper->getFragmentName().size() > 0)
-                && fragmentMgr->hasFragment(glslFragmentWrapper->getFragmentName().c_str());
+			const bool fragmentExists = (_xmlFragmentWrapper->getFragmentName().size() > 0)
+                && fragmentMgr->hasFragment(_xmlFragmentWrapper->getFragmentName().c_str());
 
 			if (!fragmentExists)
 			{
-				std::stringstream glslStream;
-				glslFragmentWrapper->getDocument(glslStream);
-				const std::string xmlFileName(Plugin::instance().getResourcePath().asString() + "/tiledImage.xml");
-				fragmentName = fragmentMgr->addShadeFragmentFromFile(xmlFileName.c_str(), false);
+                std::stringstream glslStream;
+                _xmlFragmentWrapper->getDocument(glslStream);
+                std::string xmlFileName(Plugin::instance().getResourcePath().asString() + "/standard_surface_default.xml");
+                //std::string xmlFileName(Plugin::instance().getResourcePath().asString() + "/tiledImage.xml");
+
+                // TODO: This should not be hard-coded
+                std::string dumpPath("d:/work/shader_dump/");
```

Comment:
This should probably be removed, no? Not only is it pointing to a path some people don't have, but I didn't findn the effect output directory to be particularly useful anyways.